### DISCUSSION TITLE
About the evaluation of  OpenChat-13b-V3.2 etc.

### DISCUSSION THREAD

**Discussion by niexufei** [open-compass/opencompass#342] [Q&A] [answered]
On the AlpacaEval leaderboard, models like OpenChat-13b-V3.2, WizardLM-13B-V1.2, and Vicuna-13b-V1.5-16k have shown outstanding performance. Does OpenCompass have plans to support evaluation for these models?
Thanks a lot.

**Comment by tonysy** [comment, marked as answer]
Thanks  for the suggestions, we have added these models into our evaluation plan, results will be updated in next one or two weeks.
